com.qzandronum.Q-Zandronum.appdata.xml

```diff
@@ -29,19 +29,19 @@
 
   <screenshots>
     <screenshot type="default">
-      <image>https://raw.githubusercontent.com/flathub/com.zandronum.Zandronum/master/images/image_01.png</image>
+      <image>https://raw.githubusercontent.com/flathub/com.qzandronum.Q-Zandronum/master/images/image_01.png</image>
```

Comment:
the repo doesn't exist. It's 404 Not Found

Reply:
I made that on purpose since when this gets merged, that repo will exist.
I guess ill upload the images to imgur.

Reply:
imgur is NOT a reliable way.

Reply:
> imgur is NOT a reliable way.

I will replace it when/if this gets accepted, to a github one.
It is temporary.